jr-objects/src/main/java/com/fasterxml/jackson/jr/ob/impl/SimpleValueReader.java

```diff
@@ -290,10 +293,17 @@ protected Path _readPath(JsonParser p) throws IOException {
         }
     }
 
-    protected int[] _readIntArray(JsonParser p) throws IOException
-    {
-        // !!! TODO
-        throw new JSONObjectException("Reading of int[] not yet implemented");
+    protected int[] _readIntArray(JsonParser p) throws IOException {
+        if (p.getCurrentToken().equals(JsonToken.START_ARRAY)) {
+            p.nextToken();
+        }
+
+        final IntStream.Builder builder = IntStream.builder();
+        while (!p.getCurrentToken().equals(JsonToken.END_ARRAY)) {
+            builder.add(p.getValueAsInt());
```

Comment:
I wonder what this does for `START_OBJECT` or `START_ARRAY`? We should probably do some other validation.
I'll have to think about this.